src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -1791,8 +1779,9 @@ private static unsafe int IndexOfAnyProbabilistic(ref char searchSpace, int sear
         /// </summary>
         /// <param name="span">The span to search.</param>
         /// <param name="values">The set of values to search for.</param>
-        public static int LastIndexOfAny<T>(this Span<T> span, ReadOnlySpan<T> values) where T : IEquatable<T>?
-            => LastIndexOfAny((ReadOnlySpan<T>)span, values);
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This was necessary?  Or just added for consistency with something?

Reply:
This was removed by mistake in https://github.com/dotnet/runtime/pull/75754#discussion_r1005797894

I added it back to at least make it consistent with the rest of `{Last}IndexOfAny` overloads.